eZ/Publish/API/Repository/Tests/SetupFactory/LegacyElasticsearch.php

```diff
@@ -59,7 +59,7 @@ protected function getServiceContainer()
             $containerBuilder->addCompilerPass( new Compiler\Search\Elasticsearch\AggregateSortClauseVisitorContentPass() );
             $containerBuilder->addCompilerPass( new Compiler\Search\Elasticsearch\AggregateSortClauseVisitorLocationPass() );
             $containerBuilder->addCompilerPass( new Compiler\Search\Solr\FieldRegistryPass() );
-            $containerBuilder->addCompilerPass( new Compiler\Search\Solr\SignalSlotPass() );
+            $containerBuilder->addCompilerPass( new Compiler\Search\SignalSlotPass() );
```

Comment:
May I suggest we create a `Common` namespace?
If we do this we redo the mistake done in Persistence having several folders which are not implementation of a engine at root, messes up the convention of <API/SPI-part>/Implementation organization in Core.